website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-okta.md

```diff
@@ -53,7 +53,7 @@ On the **SAML Settings** page, enter the following values:
 * **Single sign on URL**: `https://auth.getdbt.com/login/callback?connection=<your-deployment-id>`
 * **Audience URI (SP Entity ID)**: `urn:auth0:dbt-cloud:<your-deployment-id>`
 
-Replace `<your-deployment-id>` with your dbt Cloud deployment ID. If you aren't sure what value you should use, please contact support (support@getdbt.com).
+Replace `<your-deployment-id>` with your dbt Cloud deployment ID. Your deployment ID will be provided to you by your Customer Success Manager during implementation.
```

Comment:
@amychen1776 when is this provided? I am wondering if this wording clarifies enough for people. I read "id" as a number which is I think what is most confusing here but interested if you have had clients get confused on this before?